ERCS/erc-encall.md

```diff
@@ -0,0 +1,441 @@
+---
+eip: 8xxx
```

Comment:
```suggestion
eip: 8087
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.